src/test.js

```diff
@@ -733,10 +733,16 @@ function internalStart( test ) {
 }
 
 function numberOfTests( module ) {
-	var count = module.tests.length;
-	while ( ( module = module.childModule ) ) {
-		count += module.tests.length;
+	let count = module.tests.length,
+		modules = [ ...module.childModules ];
```

Comment:
Thought about this. I'd like for us to establish an ES6 convention for types of variable bindings before introducing `const` (since it is more restrictive than `let`).